integration/spark2/src/main/scala/org/apache/spark/sql/execution/command/preaaggregate/PreAggregateTableHelper.scala

```diff
@@ -126,6 +126,12 @@ case class PreAggregateTableHelper(
         newLongStringColumn.mkString(","))
     }
 
+    //Add long_string_columns properties in child table from the parent.
+    tableProperties
```

Comment:
@shardul-cr7 Here you are copying all the long String column...in case of pre aggregate data map may be it will not have all the columns of maintable, its better to add long string properties only for those column which are present in case of pre aggregate datamap......
example:  maintable long string columns: column1, column2, column3
datamap1: column1....set long string property for column1
datamap2:count(column1) no need to set if any udf, udaf is present then no need to set this property

Reply:
Done. @kumarvishal09 please review.